fastlane/lib/fastlane/new_action.rb

```diff
@@ -31,8 +31,16 @@ def self.generate_action(name)
       UI.success "Created new action file '#{path}'. Edit it to implement your custom action."
     end
 
+    def self.check_action_name_from_args(new_action_name)
+      if !new_action_name.nil? && name_valid?(new_action_name)
```

Comment:
Slightly more idiomatic might be `if new_action_name && name_valid?(new_action_name)`

Because `nil` values are false-y, it will behave as you'd like and read a little cleaner. :+1: